packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../utilities/tests/storybook';
+import { listOptionTag } from '../../list-option';
+import { numberFieldTag } from '../../number-field';
+import { selectTag } from '../../select';
+import { cardTag } from '..';
+
+const overviewText = `The \`nimble-card\` is a container that is designed to contain arbitrary content that is specified by a client
+application. The \`nimble-card\` is intended for grouping related content.`;
+
+const metadata: Meta = {
+    title: 'Incubating/Card',
+    tags: ['autodocs'],
+    parameters: {
+        docs: {
+            description: {
+                component: overviewText
+            }
+        },
+        actions: {}
+    },
+    render: createUserSelectedThemeStory(html`
```

Comment:
FYI the changes were small but wanted to see how the changes impacted the show code view. I like that the example has less custom styling / is simpler usage. I'm guessing we will want to show a bit more about sizing patterns but we'll save that for when it's closer to the finished implementation

![image](https://github.com/ni/nimble/assets/1588923/de706fb9-f4fc-4b2c-8f13-ae42098589da)